### DISCUSSION TITLE
Import choices for a CSV with separate Debit and Credit columns?

### DISCUSSION THREAD

**Discussion by guydavis** [firefly-iii/firefly-iii#8244] [Firefly III] [answered]
Hi all.  I'm trying to import from my bank which uses two different columns (both positive numbers) for payments out (debits) and payments in (credits) to the bank account.  

For example:
```
DATE,DESC,DEBIT,CREDIT
08/14/2023,E-TRANSFER OUT,25.00,
08/15/2023,PAYCHEQUE IN,,1000.00
08/15/2023,INSURANCE OUT,118.00,
```

I've tried mapping the debit/credit columns to different choices on import such as "Amount" & "Amount (negated)", and also "Amount (Credit)" & "Amount (Debit)" , however I can only have every transaction be treated as a debit or every transaction treated as a credit.  I can't get the import to record my expenses (as a minus) and my paycheque (as a positive). 

Any help would be greatly appreciated.

Thanks,
Guy

**Comment by JC5** [comment, marked as answer]
Link one column to amount (credit) and the other to amount (debit)

**Reply by guydavis** [reply]
Thanks, trying again today the import worked.
